app/models/user.rb

```diff
@@ -52,4 +52,12 @@ def approved=(val)
       self.approved_at = nil
     end
   end
+
+  def create_slide_under_event(slide_params, event_id)
```

Comment:
this method should be named as `new_slide_under_event` since it's not using `slide.save`